settings.go

```diff
@@ -88,70 +146,70 @@ func LoadSettings(ctx context.Context) (*Settings, error) {
 				defaultBaseRepo = defaultBaseRepo[:len(defaultBaseRepo)-4]
 			}
 		}
-	}
-	if maintainersCanModify == "" {
-		s.MaintainersCanModify = true
-	}
-	if err := scanner.Err(); err != nil {
-		return nil, err
-	}
-
-	// https://github.com/settings/tokens
-	if s.User == "" {
-		s.User, err = input.Ask("GitHub username", "")
-		if err != nil {
-			return nil, err
-		}
-		if s.User == "" {
-			return nil, errors.New("GitHub username required. Set `git config --global github.user $USER`")
+		if maintainersCanModify == "" {
```

Comment:
It looks like this got indented inside the `scanner.Scan()` loop accidentally

Reply:
indeed 😓 fixed!